<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelDialog</class>
 <widget class="QDialog" name="LevelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>410</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>UpArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>选择难度</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainFrame.qrc">
    <normaloff>:/new/prefix1/Image/Icon/图标.jpg</normaloff>:/new/prefix1/Image/Icon/图标.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="easyAI_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>简单人机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="middleAI_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>中等人机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="difficultAI_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>困难人机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nightmareAI_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>噩梦人机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="normalAI_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>普通人机</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>410</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Background/2_副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>easyAI_button</zorder>
  <zorder>middleAI_button</zorder>
  <zorder>difficultAI_button</zorder>
  <zorder>nightmareAI_button</zorder>
  <zorder>normalAI_button</zorder>
 </widget>
 <resources>
  <include location="MainFrame.qrc"/>
 </resources>
 <connections/>
</ui>
